client/js/components/App.react.jsx

```diff
@@ -39,6 +73,37 @@ export default class App extends React.Component {
             .tap((data) => this.setState({trailGeo: data}));
     }
 
+    setUserLocation(userLocation) {
+        var parksWithDistance = this.state.allParks.map((park) => {
+            var toPoint = turf.point([userLocation[1], userLocation[0]]);
+            var fromPoint = turf.point([park.center[1], park.center[0]]);
```

Comment:
This is going to be a bug sooner or later when we forget to do this swap. Is it not possible to tell `turf` that the data is "backwards" ?